gabby/extracts/view/gsheets_comp_events.sql

```diff
@@ -84,9 +95,11 @@ WITH
       )
       /*DSO/DCO*/
       LEFT JOIN people.staff_crosswalk_static AS d ON (l.dso_dco = d.df_employee_number)
+      LEFT JOIN campus_approval AS ca ON (l.primary_site = ca.campus_name)
       /*DSO/DCO Managers (MDSOs)*/
       LEFT JOIN people.staff_crosswalk_static AS e ON (
         d.manager_df_employee_number = e.df_employee_number
+        OR ca.mdso = e.df_employee_number
```

Comment:
can you double check that the data returned by this is the right structure? an OR on a JOIN can cause unexpected results. you might want to create a separate JOIN using the campus_approval employee_number

Reply:
yep!